Kernel/Firmware/ACPI/Parser.h

```diff
@@ -37,12 +37,14 @@ class ACPISysFSComponent : public SysFSComponent {
     virtual StringView name() const override { return m_table_name->view(); }
     virtual ErrorOr<size_t> read_bytes(off_t, size_t, UserOrKernelBuffer&, OpenFileDescription*) const override;
 
+    virtual size_t size() const override final { return m_length; }
```

Comment:
Not blocking but I noticed that we are inconsistent whether this pattern is spelled `virtual Type method() override final` or `virtual Type method() final` in the codebase overall.

Reply:
Right. Just for the sake of completeness, I'd choose `override final` which is more explicit.